cli.go

```diff
@@ -14,18 +16,20 @@ import (
 )
 
 type CLIOpts struct {
-	// TODO: CODE REMOVED
-	// MUST BE WRITTEN FROM SCRATCH WITHOUT LOOKING AT THE ORIGINAL CODE
-	// Description:
-	// Add the required variable for CLI parsing here
+	verbose     bool
+	setcap      bool
+	sinkName    string
+	loadInput   bool
+	loadOutput  bool
+	unload      bool
+	threshold   int
+	list        bool
+	checkUpdate bool
 }
 
 func parseCLIOpts() CLIOpts {
 	var opt CLIOpts
-	// TODO: CODE REMOVED
-	// MUST BE WRITTEN FROM SCRATCH WITHOUT LOOKING AT THE ORIGINAL CODE
-	// Description:
-	// Add a parameter for logs
+	flag.BoolVar(&opt.verbose, "v", false, "Verbose output (print logs to stderr)")
```

Comment:
Most programs use a `-v` or `--verbose` flag to enable info-level logging. So it's more in line with many other command-line utilities. I was initially confused by the `-log` option as I thought it would take a filename argument and write logs to that file.

I didn't think about the fact that changing this would break existing scripts when I did it. I'll change the option back to `-log` instead of `-v`.

Reply:
You don't need to change it back. I don't believe it breaks anything. If most programs use `-v`, then we'll use it.

Reply:
Done in 2d49fe1

Reply:
Perhaps it would be best to have it as `-v` or `-l`? Generally, when I think about cli opts, something with a "-" should have a single character after it, while something with a longer name has a "--". I suppose that's a long way of saying that I'm okay with this changing to `-v' or even '-l'.

Reply:
> You don't need to change it back. I don't believe it breaks anything. If most programs use `-v`, then we'll use it.

>  I suppose that's a long way of saying that I'm okay with this changing to `-v' or even '-l'.

Ok sounds good, I'll remove the 2d49fe1 and leave it as `-v`.

> Generally, when I think about cli opts, something with a "-" should have a single character after it, while something with a longer name has a "--".

I completely agree.